docs/source/concepts/AbinsDataFormats.rst

```diff
@@ -0,0 +1,100 @@
+.. _Abins Data Formats:
+
+Abins Data Formats
+==================
+
+.. list-table:: Title
```

Comment:
I think this needs to be `.. list-table::` instead of `.. list-table:: Title` as the html page looks like this for me: 
![image](https://github.com/user-attachments/assets/1f0a4a56-de3f-46ac-a415-275a53789f38)